polyfill/lib/calendar.mjs

```diff
@@ -260,8 +269,23 @@ impl['iso8601'] = {
 
 // Implementation details for Gregorian calendar
 const gre = {
-  isoYear(year, era) {
-    return era === 'bc' ? -(year - 1) : year;
+  isoYear(eraYear, era) {
+    return era === 'bc' ? -(eraYear - 1) : eraYear;
+  },
+  validateFields(fields) {
+    if (fields.eraYear === undefined && fields.year === undefined) {
+      throw new TypeError(
+        `required properties missing or undefined: must include 'year' and/or both 'era' and 'eraYear'`
```

Comment:
Oh, oops. I was running prettier with https://github.com/prettier/prettier-emacs but it differs from our apparently eslint -> prettier config. I'll update my workflow so that doesn't happen again, thank you!